docs/source/gbd2019_models/risk_risk_correlation/maternal_bmi_birthweight/index.rst

```diff
@@ -0,0 +1,131 @@
+
+Given that,
+
+.. math::
+
+   BW_\text{exposed - TMREL} = \overline{BW}_\text{exposed} - \overline{BW}_\text{TMREL}
+
+.. math::
+
+   \overline{BW}_{GBD} = p_\text{exposed} * \overline{BW}_\text{exposed} + p_\text{TMREL} * \overline{BW}_\text{TMREL}
+
+Then the difference between maternal BMI strata and the population mean from GBD can be represented as:
+
+.. math::
+
+   BW_\text{TMREL - GBD} = -p_\text{exposed} * BW_\text{exposed - TMREL}
+
+and
+
+.. math::
+
+   BW_\text{exposed - GBD} = (1 - p_\text{exposed}) * BW_\text{exposed - TMREL}
+
+Where :math:`p_\text{exposed}` represents the exposure prevalence of maternal BMI < 18.5.
+
+Therefore, the association bewteen maternal BMI risk exposure and birthweight risk exposure should be implemented according to the following steps:
+
+#. Assign maternal BMI risk exposure values as described on the :ref:` maternal BMI risk exposure page <2019_risk_exposure_maternal_bmi>`
+
+#. Assign a continuous birthweight exposure page as described on the :ref:`GBD 2019 Low birthweight short gestation risk exposure page <2019_risk_exposure_lbwsg>`
+
+#. Apply a shift to the assigned birthweight exposure value from step 2 based on the assigned maternal BMI exposure such that:
+
+.. math::
+
+   BW_\text{i, shifted} = BW_\text{i, unshifted} + BW_\text{maternal BMI exposure(i) - GBD}
+
+Causation
+++++++++++++
+
+We are not currently modeling a direct causal relationship between changes in maternal BMI exposure and changes in birthweight exposure.
+
+Assumptions and Limitations
+++++++++++++++++++++++++++++++
+
+#. We are limited in that we apply the birthweight shift between maternal BMI strata estimated from a random effects model of serveral populations from the literature to specific modeled populations even though there is evidence of significant heterogeneity between populations. 
+
+#. We are limited in that we consider only the population mean difference in birthweight between maternal BMI strata due to our dichotomous measure of maternal BMI risk exposure rather than a continuous measure of maternal BMI which would allow for a more detailed association between the two risk exposures.
+
+Validation Criteria
++++++++++++++++++++++
+
+#. The exposure distribution of birthweight in the baseline scenario should continue to validate to the GBD birthweight exposure distribution
+
+#. The difference in population mean birthweight in the exposed maternal BMI category and the TMREL maternal BMI category should approximately equal :math:`BW_\text{exposed-TMREL}`
```

Comment:
That is -138.46 grams (-174.68 to -102.25), right?

Reply:
Yes!